test/runtests.jl

```diff
@@ -131,3 +186,33 @@ end
 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 13.0; 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 8.0; 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 56470.0]], [[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 14.0, 98.0, 392.0, 1290.0, 3880.0, 10402.0, 22768.0, 46698.0, 86578.0, 146820.0, 233578.0, 346404.0, 488200.0, 649228.0, 826202.0, 1.015252e6, 1.19897e6, 1.374416e6, 1.534602e6, 1.662652e6, 1.767546e6, 1.840718e6, 1.878834e6, 1.890286e6, 1.870974e6, 1.823224e6, 1.753748e6, 1.66611e6, 1.574542e6, 1.464532e6, 1.350526e6, 1.237916e6, 1.124906e6, 1.016544e6, 904466.0, 802632.0, 708542.0, 618742.0, 539706.0, 467250.0, 400972.0, 344160.0, 293412.0, 246742.0, 208304.0, 174286.0, 144804.0, 120744.0, 100088.0, 82516.0, 66866.0, 55018.0, 44808.0, 37108.0, 29726.0, 24472.0, 20584.0, 17584.0, 14946.0, 14214.0, 13166.0, 13896.0, 14240.0, 15374.0, 17400.0, 19872.0, 23404.0, 28138.0, 33098.0, 40264.0, 47482.0, 54744.0, 65322.0, 75648.0, 88564.0, 102300.0, 117572.0, 134182.0, 152298.0, 171236.0, 192066.0, 214328.0, 235494.0, 261470.0, 287696.0, 311850.0, 337580.0, 360876.0, 386476.0, 409382.0, 430348.0, 454168.0, 471626.0, 486284.0, 501378.0, 509600.0, 513772.0, 501152.0, 460160.0, 386916.0, 287860.0, 184864.0, 103902.0, 52092.0, 23570.0, 9628.0, 3816.0, 1476.0, 444.0, 230.0, 204.0, 152.0, 214.0, 320.0, 380.0, 576.0, 876.0, 1026.0, 1452.0, 1780.0, 2344.0, 3094.0, 4046.0, 4794.0, 6446.0, 7998.0, 10642.0, 12730.0, 15780.0, 19308.0, 24944.0, 30014.0, 36732.0, 44552.0, 53062.0, 63610.0, 76170.0, 90758.0, 108690.0, 126846.0, 148398.0, 174006.0, 201100.0, 231218.0, 266426.0, 303582.0, 346028.0, 388850.0, 439320.0, 485780.0, 543498.0, 603894.0, 661394.0, 721948.0, 781186.0, 843912.0, 908314.0, 966732.0, 1.025532e6, 1.078364e6, 1.128054e6, 1.171964e6, 1.20984e6, 1.24029e6, 1.260744e6, 1.271922e6, 1.27518e6, 1.267006e6, 1.251446e6, 1.22751e6, 1.19386e6, 1.147838e6, 1.105052e6, 1.048822e6, 993920.0, 932682.0, 868794.0, 800520.0, 736902.0, 672572.0, 611724.0, 548614.0, 491772.0, 434420.0, 384892.0, 337440.0, 294738.0, 256878.0, 222366.0, 190716.0, 162536.0,
```

Comment:
Great that you started writing tests!
It would be good to move this into a data file which is read into runtest.jl and store in subfolder test_data. Have a look how this is done for the Cu55 test.